contracts/wallet/WalletFactory.sol

```diff
@@ -308,10 +198,9 @@ contract WalletFactory is Owned, Managed {
         }
         // initialise the wallet with the owner and the extended modules
         _wallet.init(_owner, extendedModules);
-        // add guardian if needed
-        if (_guardian != address(0)) {
-            IGuardianStorage(guardianStorage).addGuardian(address(_wallet), _guardian);
-        }
+        // add guardian
```

Comment:
The `(Optional)` word should be removed from the `_guardian` param in the method description.